source/docs/hardware/hardware-basics/status-lights-ref.rst

```diff
@@ -6,202 +6,511 @@ Many of the components of the FRC Control System have indicator lights that can
+|          +---------------+------------------------------+
+|          | Blue Blinking | Powering Up                  |
++----------+---------------+------------------------------+
+| Eth Link | Blue          | Link up                      |
+|          +---------------+------------------------------+
+|          | Blue Blinking | Traffic Present              |
++----------+---------------+------------------------------+
+| WiFi     | Off           | Bridge mode,                 |
+|          |               | Unlinked or non-FRC firmware |
+|          +---------------+------------------------------+
+|          | Red           | AP, Unlinked                 |
+|          +---------------+------------------------------+
+|          | Yellow/Orange | AP, Linked                   |
+|          +---------------+------------------------------+
+|          | Green         | Bridge mode, Linked          |
++----------+---------------+------------------------------+
 
 Power Distribution Panel
 ------------------------
 
-.. image:: images/status-lights/pdpLight.png
-    :width: 300
+.. image:: images/status-lights/pdp-status-lights.svg
+  :width: 600
+
++---------------------+---------------------------+
+| LED Blink/Color     | Description               |
++=====================+===========================+
+| Fast Green Blink    | Robot is enabled          |
++---------------------+---------------------------+
+| Slow Green Blink    | Robot is disabled         |
++---------------------+---------------------------+
+| Slow Orange Blink   | Robot is disabled.        |
+|                     | Sticky Fault present.     |
++---------------------+---------------------------+
+| Slow Red Blink      | No CAN Comm.              |
++---------------------+---------------------------+
+| (COMM LED only)     | Device is in boot-loader. |
+| Green/Orange Blink) | Field-upgrade necessary.  |
++---------------------+---------------------------+
+| Both LEDs off       | Device is NOT powered.    |
++---------------------+---------------------------+
 
 Voltage Regulator Module
 ------------------------
 
-.. image:: images/status-lights/vrmLight.png
-    :width: 300
+.. image:: images/status-lights/vrm-status-lights.svg
+  :width: 400
 
 The status LEDs on the VRM indicate the state of the two power supplies. If the supply is functioning properly the LED should be lit bright green. If the LED is not lit or is dim, the output may be shorted or drawing too much current.
 
 Pneumatics Control Module (PCM)
 -------------------------------
 
-.. image:: images/status-lights/pcmLight.png
-    :width: 300
-
-Solenoid Channel LEDs - These LEDs are lit red if the Solenoid channel is enabled and not lit if it is disabled.
+.. image:: images/status-lights/pcm-status-lights.svg
+  :width: 400
+
++--------+---------------+-------------------------------+------------------+
+| LED    | Strobe        | Slow                          | Long             |
```

Comment:
This table only applies to the status LED and I think we can make it clearer by adding Status to the header. I just cannot do that via the GitHub UI.

Reply:
Fixed in c172d27.